content/10.peptide-ionization.md

```diff
@@ -1,7 +1,21 @@
 ## Peptide Ionization {.page_break_before}
-
-The 2002 Nobel Prize in Chemistry was awarded to partially to John Fenn and Koichi Tanaka "for their development of soft desorption ionisation methods for mass spectrometric analyses of biological macromolecules" [@URL:https://www.nobelprize.org/prizes/chemistry/2002/summary/].
+Before MALDI and ESI[@DOI:10.1126/science.2675315], the so-called soft-ionization techniques were developed, the analysis of peptides was a challenge for mass spectrometry. 
+It was a big ask to introduce ionized peptides using a hard ionization technique (like fast atom bombardment) into the mass spectrometer, without destroying or breaking them.
+The soft ionization tehniques however, revolutionized the proteomics field and it became possible to routinely ionize and analyze peptides using MALDI and ESI techniques at high-throughput scale. 
+Such was the impact of these two techniques that the 2002 Nobel Prize in Chemistry was co-awarded to John Fenn (ESI) and Koichi Tanaka (MALDI) "for their development of soft desorption ionisation methods for mass spectrometric analyses of biological macromolecules". [@URL:https://www.nobelprize.org/prizes/chemistry/2002/summary/].
 
 ### MALDI
+In this method, the in-gel digested protein sample is co-crystallized with a matrix - a volatile, low molecular-weight, aromatic compound (like cyno-hydroxycinnamic acid).
+Subsequently, the analyte is ionized by Matrix-assisted laser desorption ionization (MALDI), in which it is subjected to a 337nm LASER.
+This laser energy is just about right to ionize the matrix and peptides without breaking them.
```

Comment:
I think this sentence is trying to comment on how much energy is used? can we be more specific and include citation(s)?

Reply:
More than energy, its inting to volatile matrix bearing the brunt of energy to dissipate their protons which effectively protects the peptides from breaking, ratio is 1000:1 :: matrix:peptides

Reply:
I like the way you phrase this comment - can you rephrase like that?